modules/ROOT/pages/test-recorder-examples.adoc

```diff
@@ -0,0 +1,66 @@
+= Recording a Test From an Exchange Example
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-partial:
+
+The following example shows you how to import into Studio an existing project from Exchange, and then record the processing of a flow to create a unit test for the flows.
+
+== Prerequesite
+
+* You have your Anypoint Platform credentials configured in Studio. +
+See xref:studio::set-credentials-in-studio-to.adoc[Configuring Anypoint Platform Credentials] for more information.
+
+== Testing the Login Form Using the HTTP Connector Example
+
+. Click the *Open Exchange* icon image:exchange-icon.png[3%,3%].
+. Select *Provided by MuleSoft* in the pop-up Exchange window, filter per *Examples* and look for the *Login Form Using the HTTP Connector*.
+. Click the *Open* button. +
+Studio imports the example as a new working project.
+** If prompted, select to *Perform Update*  in the *Update your Workspace* option.
+. Select the *Manage Mule Projects APIs* icon image:manage-project-api-icon.png[2%,2%].
+. Select *Modules* and update both *MUnit* and *Munit Tools*.
++
+image::update-munit.png[]
+. Select *Apply and Close*.
+
+=== Testing a Successful Login
+
+//tag::this-test[]
+. In your project, right-click the `DoLoginFlow`, and select > *Munit* > *Record new test for this flow (BETA)*.
++
+image::login-flow-test-recorder-example.png[]
++
+The test recorder starts your project and starts recording the event going through that flow.
+. Open a browser and navigate to `+http://0.0.0.0:8081/login+`.
+//end::this-test[]
+. In *Username* type `_mule_`, in *Password* type `_mule_`, and click *Submit*. +
+//tag::also-this[]
+The test recorder captures this event and stops the recording.
+. Back in Studio, click *Configure Test*.
++
+image::stops-recording.png[]
+. Select the file name for the MUnit test suite in which you want to record your test.
+//end::also-this[]
+. Under *Test name*, type `_DoLoginFlow-successfull_`
+. Set the test recorder to set the recorded input of the flow, and assert that the recorded output is the expected output by the test.
+. In the *Successful login* parse template step, select *Verify that this process was called* and set the counter to *equal to* `_1_` time.
+. Click *Next* and then *Finish*.
++
+A fully functional MUnit test is generated to assert that the process in your application works as expected when the login is successful.
+
+=== Testing an Unsuccessful Login
```

Comment:
Maybe the titles should be "Testing the Process for a Successful[an Unsuccessful] Login Attempt"?